src/main/java/com/openshift/restclient/model/limit/ILimit.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors: Red Hat, Inc.
+ ******************************************************************************/
+package com.openshift.restclient.model.limit;
+
+/**
+ * @author HyunsooKim1112
+ */
+public interface ILimit {
+	
+	public String getType();
+	
+	public void setType(String type);
+		
+	public String getMaxMemory();
+	
+	public void setMaxMemeory(String maxMemory);
```

Comment:
Spelling.  Also, we have another pending PR that handles some memory conversions.  I wonder if these should be modified to take advantage of that code.